### DISCUSSION TITLE
Chart control

### DISCUSSION THREAD

**Discussion by louisoft88** [CommunityToolkit/Labs-Windows#575] [Proposals]
I miss a general-purpose, multi-axes chart control with basic charts, Column, bar (stacking), line. scatter etc... Am I the only one?

**Comment by Arlodotexe** [comment]
You might be looking for something like https://github.com/ScottPlot/ScottPlot?

**Comment by hawkerm** [comment]
Links to past discussions on the topic:
- https://github.com/CommunityToolkit/WindowsCommunityToolkit/issues/563
- https://github.com/microsoft/microsoft-ui-xaml/issues/1794

Other OSS Charting Options:
- https://github.com/ScottPlot/ScottPlot (as mentioned above)
- https://github.com/Live-Charts/LiveCharts2
- https://github.com/microcharts-dotnet/Microcharts
- https://github.com/HavenDV/H.OxyPlot
  - https://xamlbrewer.wordpress.com/2022/02/28/drawing-charts-and-diagrams-with-oxyplot-in-winui-3/

Commercial:
- https://www.syncfusion.com/winui-controls
  - https://help.syncfusion.com/winui/cartesian-charts/overview
- https://www.telerik.com/winui/chart
  - https://docs.telerik.com/devtools/winui/controls/radchart/overview
- https://developer.mescius.com/componentone/winui-controls/flexchart-winui-chart-controls
- https://www.devexpress.com/winui/
- https://lightningchart.com/net-charts/ (UWP Only Still)

Prototypes:
- https://github.com/xyzzer/Win2DChartSample
- https://github.com/Romasz/DXCharts
- https://github.com/apchee/Chart.Win2D.WinUI3

**Reply by michael-hawker** [reply]
Cross-linking to WinUI issue on this topic here: https://github.com/microsoft/microsoft-ui-xaml/issues/1794 (Edit: Oh that only works for issues, and I have this above, boo.)
